resources/views/Item/Components/SingleItem.twig

```diff
@@ -118,6 +118,8 @@
                         <a {% if shippingCat is not empty %} data-toggle="modal" href="#shippingscosts"{% endif %} title="{{ trans("Ceres::Template.itemShippingCosts") }}">{{ trans("Ceres::Template.itemShippingCosts") }}</a>
                     </span>
 
+                    <slot name="AdditionalContentAfterVAT"></slot>
```

Comment:
du definierst hier zwar den slot aber du hast vergessen in der SingleItemView.twig auch den LayoutContainer hinzuzufügen